src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <p>
+          {' Posted by  '}
+          <UserInfo user={user} />
```

Comment:
user can be null, let's add the check
<img width="537" alt="image" src="https://github.com/user-attachments/assets/75786daf-2953-4356-930a-76e2c8602977">
```suggestion
          {user && <UserInfo user={user} />}
```